.circleci/config.yml

```diff
@@ -639,23 +653,41 @@ workflows:
             branches:
               only: env/dev2
 
-      - hodl:
+      - hodl_blue:
           type: approval
           filters:
             branches:
               ignore: /.*/
             tags:
               only: /^v.*$/
 
-      - deploy_uat:
+      - hodl_green:
+          type: approval
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /^v.*$/
+
+      - deploy_uat_blue:
           requires:
             - eunit
             - static_analysis
             - linux_package
-            - hodl
+            - hodl_blue
           filters:
             branches:
-              ignore: /.*/
+              only: env/uat_blue
```

Comment:
How can this work - considering that required job `hodl_blue` is not triggered on push to branch `env/uat_blue`?

Reply:
see 1e3363e